tests/cases/employee_health_portal/case_employee_health_portal_account.yaml

```diff
@@ -316,4 +316,94 @@ TestEmployeeHealthPortalAccountHelp:
           Value: Employee_Health_User_Manual
         - Type: contain
           Var: HELP_PAGE_URL
-          Value: .pdf
\ No newline at end of file
+          Value: .pdf
+#CXTB-109 Employee Health Portal - Request an Appointment for User
+TestEmployeeHealthPortalRequestAnAppointmentForUser:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  #Calling Login case
+    - Type: case
+      Value: EmployeeHealthPortalLogIn
+  #Going to Request an Appointment
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.request_appointment_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.form_modal$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.create_event_button$
+  # Checking error message
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.error_message$
+  # closing form
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.cancel_button$
+  # waiting for appointment view
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.request_appointment_button$
+  # filling form
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.select_location_drop_down$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.select_location_option$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.select_patient_drop_down$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.select_patient_option$
```

Comment:
Test is failing finding this element

Reply:
fixed. this page object have been deleted because it is not required.